src/paymentservice/package.json

```diff
@@ -16,15 +16,15 @@
     "@google-cloud/trace-agent": "5.1.6",
     "@grpc/grpc-js": "1.4.4",
     "@grpc/proto-loader": "0.5.6",
-    "@opentelemetry/api": "^1.0.3",
+    "@opentelemetry/api": "^1.1.0",
     "@opentelemetry/exporter-collector-grpc": "^0.25.0",
```

Comment:
can `opentelemetry/exporter-collector-grpc` be removed?

Reply:
Yes. I forgot about removing that one. Thanks for the catch.